secretmanager/src/main/java/secretmanager/AccessSecretVersion.java

```diff
@@ -45,6 +47,15 @@ public static void accessSecretVersion(String projectId, String secretId, String
       // Access the secret version.
       AccessSecretVersionResponse response = client.accessSecretVersion(secretVersionName);
 
+      // Verify checksum.
+      byte[] data = response.getPayload().getData().toByteArray();
+      Checksum checksum = new CRC32C();
+      checksum.update(data, 0, data.length);
```

Comment:
nit: consider making this a single line (same with the other example)

Reply:
Sorry, I must not have realized that the checksum was a void return type - this is totally fine.